File: launchdarkly/php/src/GetLeadTimeChartExample.php
<?php

namespace OSEG\LaunchDarklyExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use LaunchDarkly;

$config = LaunchDarkly\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("Authorization", "YOUR_API_KEY");

try {
    $response = (new LaunchDarkly\Client\Api\InsightsChartsBetaApi(config: $config))->getLeadTimeChart(
        project_key: "projectKey_string",
        environment_key: null,
        application_key: null,
        from: null,
        to: null,
        bucket_type: null,
        bucket_ms: null,
        group_by: null,
        expand: null,
    );

    print_r($response);
} catch (LaunchDarkly\Client\ApiException $e) {
    echo "Exception when calling InsightsChartsBetaApi#getLeadTimeChart: {$e->getMessage()}";
}

File: launchdarkly/python/src/GetLeadTimeChartExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.InsightsChartsBetaApi(api_client).get_lead_time_chart(
            project_key=None,
            environment_key=None,
            application_key=None,
            var_from=None,
            to=None,
            bucket_type=None,
            bucket_ms=None,
            group_by=None,
            expand=None,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling InsightsChartsBetaApi#get_lead_time_chart: %s\n" % e)
